core/trino-main/src/test/java/io/trino/server/security/oauth2/TestingHydraIdentityProvider.java

```diff
@@ -324,7 +325,7 @@ public static void main(String[] args)
                                     .put("http-server.https.keystore.path", Resources.getResource("cert/localhost.pem").getPath())
                                     .put("http-server.https.keystore.key", "")
                                     .put("http-server.authentication.type", "oauth2")
-                                    .put("http-server.authentication.oauth2.issuer", authServerUrl)
+                                    .put("http-server.authentication.oauth2.issuer", ISSUER)
```

Comment:
What is the difference between these two? Can we remove `authServerUrl` variable?

Reply:
Not really. `ISSUER` is a value which ends up in a token's claim which Trino validates. This means that whatever `URLS_SELF_ISSUER` value is, it has to be equal to "http-server.authentication.oauth2.issuer". In most cases it would be a substring of `http-server.authentication.oauth2.auth-url`. Not in this case however. Hydra container is not aware of it's port prior to the start so it's using an url which is not accessible from the host.